engine/scan/index.md

```diff
@@ -116,6 +116,24 @@ Licenses:          enabled
 Note that we do not currently have vulnerability data for your image.
 ```
 
+## Using an HTTP proxy
+
+If your computer needs to use an HTTP proxy, then set the `HTTPS_PROXY` environment variable when using scan. For example:
+
+```console
+$ export HTTPS_PROXY=http://username:password@proxy:port
```

Comment:
How about `address` or `proxyAddress` instead of `proxy`? Both here and for powershell.
So:
```suggestion
$ export HTTPS_PROXY=http://username:password@proxyAddress:port
```